docs/docs/cmd/spo/web/web-applicationcustomizer-add.md

```diff
@@ -0,0 +1,56 @@
+# spo web applicationcustomizer add
+
+Add an application customizer to a site.
+
+## Usage
+
+```sh
+m365 spo web applicationcustomizer add [options]
+```
+
+## Options
+
+`-t, --title <title>`
+: The title of the Application Customizer.
+
+`-u, --webUrl <webUrl>`
+: URL of the site for which to retrieve the information.
+
+`-i, --clientSideComponentId <clientSideComponentId>`
+: The Client Side Component Id (GUID) of the application customizer.
+
+`--clientSideComponentProperties <clientSideComponentProperties>`
+: The Client Side Component properties of the application customizer.
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+Running this command from the Windows Command Shell (cmd.exe) or PowerShell for Windows OS XP, 7, 8, 8.1 without bash installed might require additional formatting for command options that have JSON, XML or JavaScript values because the command shell treat quotes differently. For example, this is how ApplicationCustomizer user custom action can be created from the Windows cmd.exe:
```

Comment:
We shouldn't mention the custom action here, because we're talking specifically about application customizer